### PR TITLE
Emlyn

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: ExamenPGG.Business/Bootup/Bootup.cs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+            Console.ReadKey();
+
+            isReady = true;
+
+        }
+
+        public void CreateGame()
+        {
+
```
Comment (ghost) [inline ExamenPGG.Business/Bootup/Bootup.cs:105]: Currently unused:
Boot sequence in main program works,
but if copypated into this method, it reads 
Player newPlayer = new Player()
as if Player is a namespace only?

File: ExamenPGG.Business/Bootup/IBootup.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+namespace ExamenPGG.Business.Bootup
+{
+    public interface IBootup
+    {
+        string AskPlayerNumber();
+        void InitialPlayerNumber();
+        string AskPlayerNames();
+        void InputPlayerNames();
+        void CreateGame();
```
Comment (ghost) [inline ExamenPGG.Business/Bootup/IBootup.cs]: CreateGame()
currently unused

File: ExamenPGG.Business/Logging/Logger.cs [hunk 3]
```diff
@@ -0,0 +1,21 @@
+﻿using ExamenPGG.Business.Game;
```
Comment (ghost) [inline ExamenPGG.Business/Logging/Logger.cs]: Added logger class, was needed to setup boot sequence

**Review by ghost — COMMENTED** [review]
Added comments

File: ExamenPGG.Business/Game/GameBoard.cs [hunk 4]
```diff
@@ -6,7 +6,7 @@ public class GameBoard : IGameBoard
     {
         private static GameBoard _instance;
 
-        private GameBoard()
+        public GameBoard()
```
Comment (E-Verkimpe) [inline ExamenPGG.Business/Game/GameBoard.cs]: constructor van gameboard MOET private blijven.
anders is het geen singleton patroon meer. bedoeling is dat je er maar 1 instantie van kan maken.

File: ExamenPGG.Business/Bootup/IBootup.cs [hunk 5]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (E-Verkimpe) [inline ExamenPGG.Business/Bootup/IBootup.cs]: ongebruikte using statements

**Review by KeithVanW — APPROVED** [review]

**Review by matstes — APPROVED** [review]
looks okay

**Review by E-Verkimpe — APPROVED** [review]

**Pull request merged**